mappings/net/minecraft/world/biome/source/util/VanillaTerrainParameters.mapping

```diff
@@ -1,23 +1,85 @@
 CLASS net/minecraft/unmapped/C_vtphaipm net/minecraft/world/biome/source/util/VanillaTerrainParameters
 	FIELD f_ckpicmcx offset Lnet/minecraft/unmapped/C_cjntsjzo;
+	FIELD f_jhznnevt CUBIC_SPLINE_CODEC Lcom/mojang/serialization/Codec;
 	FIELD f_lzfscfwv CODEC Lcom/mojang/serialization/Codec;
 	FIELD f_nbpisubo GLOBAL_OFFSET F
 	FIELD f_vsidgzvf SPLINE_CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_wgilnbpy NO_TRANSFORM Lnet/minecraft/unmapped/C_oqsszdfo;
 	FIELD f_wnivazcp jaggedness Lnet/minecraft/unmapped/C_cjntsjzo;
 	FIELD f_xggrdyai factor Lnet/minecraft/unmapped/C_cjntsjzo;
 	METHOD <init> (Lnet/minecraft/unmapped/C_cjntsjzo;Lnet/minecraft/unmapped/C_cjntsjzo;Lnet/minecraft/unmapped/C_cjntsjzo;)V
 		ARG 1 offset
 		ARG 2 factor
 		ARG 3 jaggedness
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 o
+	METHOD m_bznxzqmc buildJaggedErosionSpline (FFFFLnet/minecraft/unmapped/C_oqsszdfo;)Lnet/minecraft/unmapped/C_cjntsjzo;
+		ARG 4 floatFunction
+	METHOD m_dxecsmso buildErosionOffsetSpline (FFFFFFZZLnet/minecraft/unmapped/C_oqsszdfo;)Lnet/minecraft/unmapped/C_cjntsjzo;
+		ARG 0 value
+		ARG 1 value2
+		ARG 2 value3
+		ARG 3 value4
+		ARG 4 value5
+		ARG 5 value6
+		ARG 8 floatFunction
+	METHOD m_ffzeiruy factor (Lnet/minecraft/unmapped/C_vtphaipm$C_wfilltqw;)F
+		ARG 1 noise
+	METHOD m_hfnzwxim offset (Lnet/minecraft/unmapped/C_vtphaipm$C_wfilltqw;)F
+		ARG 1 noise
+	METHOD m_hsxbmunf peaksAndValleys (F)F
+		ARG 0 weirdness
+	METHOD m_htxzicvn calculateSlope (FFFF)F
+	METHOD m_jgcvxkpg mountainContinentalness (FFF)F
```

Comment:
Should this be `getMountainContinentalness`?

Currently it's a method with an adjective name rather than a verb name.

Reply:
Iirc this just does some mathematical functions, so `computeMountainContinentalness` would make more sense imo